test/e2e/main_test.go

```diff
@@ -83,17 +86,28 @@ func main(m *testing.M) int {
 
 func setupFramework() error {
 	cfg := config.GetConfigOrDie()
+	scheme := operator.NewScheme(&operator.OperatorConfiguration{})
+	err := configv1.AddToScheme(scheme)
```

Comment:
The doc says that AddToScheme is deprecated.
```suggestion
	err := configv1.Install(scheme)
```